src/BreadcrumbItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import SafeAnchor from './SafeAnchor';
+
+const BreadcrumbItem = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    linkId: React.PropTypes.string,
+    href: React.PropTypes.string,
+    title: React.PropTypes.node,
+    target: React.PropTypes.string
+  },
+
+  render() {
+    const {
+      href,
+      title,
+      target,
+      linkId,
+      children,
+      ...props } = this.props;
+    const active = (href == null);
```

Comment:
I don't think we necessarily need to deal with those - just don't render the anchor for active components. Otherwise, maybe it makes more sense to spread the input props onto the anchor rather than the `li`? e.g. if the user wants to specify `onClick` or something.